testsuite/features/secondary/srv_dist_channel_mapping.feature

```diff
@@ -25,8 +25,8 @@ Feature: Distribution Channel Mapping
     Then I should see a "Create Distribution Channel Map" text
     When I enter "SUSE Linux Enterprise Server 15 SP 4" as "os"
     And I enter "15.5" as "release"
-    And I select "x86_64" from "architecture" dropdown
-    And I select "SLE-Product-SLES15-SP4-Pool for x86_64" from "channel_label" dropdown
+    And I select "x86_64" from "architecture"
+    And I select "SLE-Product-SLES15-SP4-Pool for x86_64" from "channel_label"
```

Comment:
It would be better to hide these technical details inside new step definitions too.
And personally, I think that from the user point of view, knowing that we are acting on a dropdown webelement can help.

Reply:
In that case, do you want to add this to all other tests that use dropdowns? We do the same for dropdowns everywhere else: `And I select "equals" from "matcher"`, `And I select "1-centos7_ssh_minion_key" from "activationKeys"` etc. Currently only the steps with Select2 were different, this change makes it be the same way everywhere. Unless I misunderstand what you mean?

Reply:
@srbarrios ping.

Reply:
Hi @Etheryte the Gherkin syntax and Cucumber scenarios have been designed to use natural language to describe the steps a user must follow to achieve something. Ideally, we should write the scenarios as if were on the official documentation. So, even if as you pointed out, other steps don't follow this good practice, let's try to do it properly wherever we can.

Reply:
Or see my proposal above.